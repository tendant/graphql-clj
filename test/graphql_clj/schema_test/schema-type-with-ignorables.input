        # ignored first line and comments
# and, another comment line
	# and yet another
type# no space before comment
Image # space before comment
{width:Int!height# no spaces between tokens (this is line 6)
	:
   Int	! # previous line ends with a CR LF, this should be line 8
	alt : # tab before alt should put it in column 8
       	String # 7 spaces + tab before String should put it at column 8
        	aspect:Float # 8 spaces + tab before String should put it at column 16
bpp:	Int	! # 2 CR after newline should put this at line 12+2 = 14
# comments before closing
# the type
 } # at line 17
