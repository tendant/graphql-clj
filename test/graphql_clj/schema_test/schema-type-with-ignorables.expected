{:objects
 {:Image
  {:fields
   {:width {:type (non-null Int)},
    :height {:type (non-null Int)},
    :alt {:type String},
    :aspect {:type Float},
    :bpp {:type (non-null Int)}}}}}
